api/src/test/java/org/pmiops/workbench/workspaces/WorkspaceOperationMapperTest.java

```diff
@@ -87,9 +87,7 @@ public void test_toModelWithWorkspace() {
     DbWorkspace dbWorkspace =
         workspaceDao.save(
             new DbWorkspace().setWorkspaceNamespace("a").setName("b").setFirecloudName("c"));
-    Workspace expectedWorkspace = workspaceMapper.toApiWorkspace(dbWorkspace);
```

Comment:
refactored to stop using this unsafe version of `toApiWorkspace()` which I have also renamed `onlyForMappingRecentWorkspace()`